pkg/arvo/sys/hoon.hoon

```diff
@@ -1897,6 +1898,1491 @@
     ?~  a  ~
     ?~(r.a [~ n.a] $(a r.a))
   --
+::                                                      ::
+::::  2r: psq logic                                     ::
```

Comment:
it might be nice to expand "psq" to "priority search queue," at least in the documentation comments, or otherwise find a nice short english name for it

Reply:
:+1: I'll touch up the comments in general.

Reply:
Some comments added in 58d0ec8.